tools/generator/src/Generator+AddTargets.swift

```diff
@@ -156,26 +156,36 @@ Product for target "\(id)" not found in `products`
             shellScript: #"""
 set -eu
 
+if [ "$ACTION" == "indexbuild" ]; then
+  output_base="$OBJROOT/bazel_output_base"
+fi
+
 output_path=$(\#(bazelExec) \
+  ${output_base:+--output_base "$output_base"} \
   info \
+  ${output_base:+--experimental_convenience_symlinks=ignore} \
```

Comment:
Since we don't use the convenience links in either version of this build, I could do this unconditionally?